bindings/python/examples/how_tos/native_tokens/mint.py

```diff
@@ -0,0 +1,54 @@
+from iota_sdk import Wallet, utf8_to_hex
+from dotenv import load_dotenv
+import time
+import os
+
+load_dotenv()
+
+# In this example we will mint native tokens
+
+wallet = Wallet('./alice-database')
+
+account = wallet.get_account('Alice')
+
+if 'STRONGHOLD_PASSWORD' not in os.environ:
+    raise Exception(".env STRONGHOLD_PASSWORD is undefined, see .env.example")
+
+wallet.set_stronghold_password(os.environ["STRONGHOLD_PASSWORD"])
+
+# Sync account with the node
+response = account.sync()
+
+print('Preparing alias output transaction...')
+
+transaction = account.prepare_create_alias_output(None, None).send()
+print(f'Transaction sent: {transaction["transactionId"]}')
+
+# Wait for transaction to get included
+blockId = account.retry_transaction_until_included(transaction['transactionId'])
+print(f'Block included: {os.environ["EXPLORER_URL"]}/block/{blockId}')
+
+account.sync()
+print("Account synced")
+
+print('Preparing minting transaction...')
+
+params = {
+    "circulatingSupply": hex(100),
+    "maximumSupply": hex(100),
+    "foundryMetadata": utf8_to_hex('Hello, World!'),
```

Comment:
How about something like this?
```suggestion
    "foundryMetadata": utf8_to_hex('Here could be your https://github.com/iotaledger/tips/blob/main/tips/TIP-0030/tip-0030.md'),
```

Reply:
This is something I would prefer to have in the docs. Easier to maintain and automatically check there.

But what bothers me even more: You are not using a wiki link :sadness:

(Will add the TIP to the doc)

Reply:
Wdyt? https://github.com/iotaledger/iota-sdk/pull/541/commits/8fec5545eb91565fc43a2513e4f276c2e74aa70b

Reply:
The extra spaces there are useless, but otherwise it's fine :trollface:

Reply:
They are to mark the new line tho ;)